### PR TITLE
:construction_worker: Konflux migration

### PR DESCRIPTION
**Pull request by chassing**
* :construction_worker: Konflux migration
* Glitchtip v4.1.5 upgrade

Tickets:
* [APPSRE-10967](https://issues.redhat.com/browse/APPSRE-10967)
* [APPSRE-11006](https://issues.redhat.com/browse/APPSRE-11006)

### CODE REVIEW

File: .tekton/glitchtip-main-pull-request.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  - name: output-image
+    value: quay.io/redhat-user-workloads/app-sre-tenant/glitchtip-main/glitchtip-main:on-pr-{{revision}}
+  - name: image-expires-after
+    value: 100d
+  - name: dockerfile
+    value: Dockerfile
+  - name: path-context
+    value: .
+  - name: target-stage
+    value: prod
```
Comment (fishi0x01) [inline .tekton/glitchtip-main-pull-request.yaml:32]: maybe test here?

Comment (fishi0x01) [reply .tekton/glitchtip-main-pull-request.yaml:32]: Ah there are no test stages in the dockerfile. Forget what I said :D

Comment (chassing) [reply .tekton/glitchtip-main-pull-request.yaml:32]: Tests will be done via an acceptance test during the deployment

**Review by fishi0x01 — APPROVED** [review]

**Pull request merged**